qiskit_ibm_runtime/constants.py

```diff
@@ -23,10 +23,21 @@
 QISKIT_IBM_RUNTIME_API_URL = "https://auth.quantum-computing.ibm.com/api"
 
 API_TO_JOB_STATUS = {
+    "CREATING": JobStatus.INITIALIZING,
+    "CREATED": JobStatus.INITIALIZING,
+    "TRANSPILING": JobStatus.INITIALIZING,
+    "TRANSPILED": JobStatus.INITIALIZING,
+    "VALIDATING": JobStatus.VALIDATING,
+    "VALIDATED": JobStatus.VALIDATING,
     "QUEUED": JobStatus.QUEUED,
+    "PENDING_IN_QUEUE": JobStatus.QUEUED,
     "RUNNING": JobStatus.RUNNING,
     "COMPLETED": JobStatus.DONE,
     "FAILED": JobStatus.ERROR,
+    "ERROR_CREATING_JOB": JobStatus.ERROR,
+    "ERROR_VALIDATING_JOB": JobStatus.ERROR,
+    "ERROR_RUNNING_JOB": JobStatus.ERROR,
+    "ERROR_TRANSPILING_JOB": JobStatus.ERROR,
```

Comment:
We don't get these additional statuses anymore. They were from the old Qobj path.

Reply:
Okay, removed the extra values.